spec/ruby/language/range_spec.rb

```diff
@@ -28,3 +28,16 @@
     end
   end
 end
+
+describe "Object Ranges" do
+  ruby_version_is "3.0" do
```

Comment:
Could you add a spec that `Range.new(1, 2)` is frozen on 3+? That seems missing above.